.prettierignore

```diff
@@ -1,5 +1,8 @@
 target
-languages
+languages/*
+!/languages/kotlin
+languages/kotlin/*
```

Comment:
Is this needed?

Reply:
It's a bit awkward but that seems to be how the ignore file works yeah:
https://github.com/prettier/prettier/issues/3328#issuecomment-350732773

We could always remove it and add an extra step on the Readme to format the doc file after it's generated.